tools/migrate-assets-to-screenly.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+
+import click
+import os
+import requests
+import sys
+import traceback
+
+from inspect import cleandoc
+from requests.auth import HTTPBasicAuth
+from requests.exceptions import RequestException
+from tenacity import retry
+
+HOME = os.getenv('HOME')
+
+BASE_API_SCREENLY_URL = 'https://api.screenlyapp.com'
+ASSETS_ANTHIAS_API = 'http://127.0.0.1/api/v1.1/assets'
+
+PORT = 80
+
+token = None
+
+
+#############
+# Utilities #
+#############
+
+def progress_bar(count, total, text=''):
+    """
+    This simple console progress bar
+    For display progress asset uploads
+    """
+    progress_line = "\u2588" * int(round(50 * count / float(total))) + '-' * (50 - int(round(50 * count / float(total))))
+    percent = round(100.0 * count / float(total), 1)
+    sys.stdout.write('[%s] %s%s %s\r' % (progress_line, percent, '%', text))
+    sys.stdout.flush()
+
+
+def set_token(value):
+    global token
+    token = 'Token %s' % value
+
+
+############
+# Database #
+############
+
+def get_assets_by_anthias_api():
+    if click.confirm('Do you need authentication to access Anthias API?'):
+        login = click.prompt('Login')
+        password = click.prompt('Password', hide_input=True)
+        auth = HTTPBasicAuth(login, password)
+    else:
+        auth = None
+    response = requests.get(ASSETS_ANTHIAS_API, timeout=10, auth=auth)
+
+    response.raise_for_status()
+    return response.json()
+
+
+############
+# Requests #
+############
+
+@retry
+def get_post_response(endpoint_url, **kwargs):
+    return requests.post(endpoint_url, **kwargs)
+
+def send_asset(asset):
+    endpoint_url = '%s/api/v4/assets' % BASE_API_SCREENLY_URL
+    headers = {
+        'Authorization': token,
+        'Prefer': 'return=representation'
+    }
+    asset_uri = asset['uri']
+
+    if asset_uri.startswith('/data'):
+        asset_uri = os.path.join(HOME, 'screenly_assets', os.path.basename(asset_uri))
+
+    data = {
+        'title': asset['name'],
+        'source_url': asset_uri
```

Comment:
You should not set source_url if the asset is a local file, I think it's probably okay to leave it because the field will be ignored, but just to be aware.